PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.h

```diff
@@ -125,7 +125,7 @@ class ProjectPanel : public DockingDlgInterface {
 	void recursiveAddFilesFrom(const TCHAR *folderPath, HTREEITEM hTreeItem);
 	HTREEITEM addFolder(HTREEITEM hTreeItem, const TCHAR *folderName);
 
-	bool writeWorkSpace(const TCHAR *projectFileName = NULL);
+	bool writeWorkSpace(const TCHAR *projectFileName = NULL, bool updateName = true);

```

Comment:
Please use `doUpdateGUI` instead of "updateName".

Reply:
Made the change as suggested.